pkg/util/mount/mount_windows.go

```diff
@@ -362,10 +362,23 @@ func (mounter *SafeFormatAndMount) formatAndMount(source string, target string,
 	glog.V(4).Infof("Attempting to formatAndMount disk: %s %s %s", fstype, source, target)
 
 	if err := ValidateDiskNumber(source); err != nil {
-		glog.Errorf("azureMount: formatAndMount failed, err: %v\n", err)
+		glog.Errorf("diskMount: formatAndMount failed, err: %v\n", err)
 		return err
 	}
 
+	if len(fstype) == 0 {
+		// Use 'NTFS' as the default
+		fstype = "NTFS"
```

Comment:
when defaulting to NTFS, would it be more robust to validate options too (in case a wrong option is passed)?

Reply:
@rootfs I would not do fstype check here, if user specify a wrong fstype, then mount error will return. So if there is new fstype is available, then we don't need to change the code to support new fstype.